mirrord/config/src/feature/network/incoming.rs

```diff
@@ -163,6 +162,61 @@ impl MirrordToggleableConfig for IncomingFileConfig {
     }
 }
 
+impl<'de> Deserialize<'de> for IncomingFileConfig {
```

Comment:
Manual deserialization makes me sad, why do we need this?

Reply:
Errors, untagged enum errors are so useless